nav/stepper/stepper_constants.py

```diff
@@ -20,9 +22,10 @@
 FL_step = 24
 FL_dir = 23
 
-CW = 1 			                 # Clockwise rotation
-CCW = 0 		                 # Counter-clockwise rotation
-STEPS_PER_REVOLUTION = 800 		 # Steps per Revolution (360/1.8) * 4. Multiply by 4 because quarter step.
+CW = 1                           # Clockwise rotation
+CCW = 0                          # Counter-clockwise rotation
+STEPS_PER_REVOLUTION = 800       # Steps per Revolution (360/1.8) * 4. Multiply by 4 because quarter step.
 DISTANCE = 60*math.pi/25.4
 STEPS_PER_INCH = STEPS_PER_REVOLUTION/DISTANCE
 STEPPER_DELAY = 0.0005
+STEPS_PER_INCH_STRAFE = STEPS_PER_REVOLUTION/(DISTANCE*0.9)  # Calibrates steps_per_inch for strafe motion.
```

Comment:
This calibration needs to happen for every single wheel / chassis design. It need to be more accessible and clear for an average user. Get together with @BenjiLee0002 to figure out proper structure.

Reply:
This will be addressed in later pull request